app/models/custom_filter.rb

```diff
@@ -17,8 +17,23 @@
 class CustomFilter < ApplicationRecord
   self.ignored_columns += %w(whole_word irreversible)
 
-  alias_attribute :title, :phrase
-  alias_attribute :filter_action, :action
```

Comment:
I am not familiar with `alias_attribute` but wouldn't `alias` for each method work?

Reply:
No, because `alias` copies already-declared methods. In this case, `phrase` and `action` are not defined methods at the moment of class declaration.